capstone-backend/app.js

```diff
@@ -18,27 +18,27 @@ Parse.serverURL = 'http://parseapi.back4app.com/'
 
 async function getUserInfo(user){
     const Movie = Parse.Object.extend("Movie");
-    const movie_query = new Parse.Query(Movie);
+    const movieQuery = new Parse.Query(Movie);
 
-    movie_query.equalTo("User", user);
-    movie_query.equalTo("active", true);
-    const user_movies = await movie_query.find();
+    movieQuery.equalTo("User", user);
+    movieQuery.equalTo("active", true);
+    const userMovies = await movieQuery.find();
 
     const Show = Parse.Object.extend("Show");
-    const show_query = new Parse.Query(Show);
-    show_query.equalTo("User", user);
-    show_query.equalTo("active", true);
-    const user_shows = await show_query.find();
+    const showQuery = new Parse.Query(Show);
+    showQuery.equalTo("User", user);
+    showQuery.equalTo("active", true);
```

Comment:
The "active" field is true if that movie is currently in the list of a that user's added movies. It becomes false if the user removes the movie. The reason I created the "active" field is because removing a movie from the database (using object.destroy()) took a long time to process, so instead I used the "active" field to determine whether to display a movie/show/hobby or not.  I just removed the active field and changed it back to removing the movie/show/hobby from the table when the user removes it. I'll figure out a solution to the long wait time later.